.github/workflows/linux-ppc64le.yml

```diff
@@ -0,0 +1,64 @@
+name: Native ppc64le Linux Test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  native_ppc64le:
+    # To enable this workflow on a fork, comment out:
+    if: github.repository == 'numpy/numpy'
+    runs-on: ubuntu-24.04-ppc64le
+    
+    name: "Native PPC64LE"
+    steps:
+    
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+        fetch-tags: true
+        persist-credentials: false
+
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y python3 python3-pip python3-dev cmake ninja-build gfortran \
+                            build-essential libopenblas-dev liblapack-dev pkg-config
+        pip install --upgrade pip
+        pip install -r requirements/build_requirements.txt
+        pip install pytest pytest-xdist hypothesis typing_extensions pytest-timeout
+
+    - name: Install spin
+      run: |
+        python3 -m pip install -U pip setuptools wheel
+        python3 -m pip install spin
+        echo "/home/runner/.local/bin" >> $GITHUB_PATH
+
+    - name: Configure Git safe directory
+      env:
+        HOME: /tmp
+      run: |
+        git config --global --add safe.directory "$PWD"
+
+    - name: Meson Build
+      run: |
+        spin build --clean -- -Dallow-noblas=true
+
+    - name: Meson Log
+      if: always()
+      run: cat build/meson-logs/meson-log.txt
+
+    - name: Run Tests
+      run: |
+        export F90=$(which gfortran)
+        spin test -- --timeout=600 --durations=10
```

Comment:
Why is the timeout value so high? If a test is that slow, I think it needs investigating.